From: Stephen Smalley <sds@tycho.nsa.gov>
Date: Fri, 27 Feb 2015 16:23:59 -0500
Subject: [PATCH] security/yama: Remove unnecessary selects from Kconfig.

Yama selects SECURITYFS and SECURITY_PATH, but requires neither.
Remove them.

Signed-off-by: Stephen Smalley <sds@tycho.nsa.gov>
---
 security/yama/Kconfig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/security/yama/Kconfig b/security/yama/Kconfig
index 20ef5143c0c0..3123e1da2fed 100644
--- a/security/yama/Kconfig
+++ b/security/yama/Kconfig
@@ -1,8 +1,6 @@
 config SECURITY_YAMA
 	bool "Yama support"
 	depends on SECURITY
-	select SECURITYFS
-	select SECURITY_PATH
 	default n
 	help
 	  This selects Yama, which extends DAC support with additional
